libtenzir/builtins/operators/python.cpp

```diff
@@ -468,22 +468,42 @@ class plugin final : public virtual operator_plugin<python_operator>,
 
   auto initialize(const record& plugin_config, const record& global_config)
     -> caf::error override {
+    auto is_node = try_get_or<bool>(global_config, "tenzir.is-node", false);
+    if (!is_node) {
+      return is_node.error();
+    }
```

Comment:
This will incorrectly fail when you run the Python operator remotely by manually using `remote python "…"`.

What you can do instead is to add the following to the operator's implementation to force it to run at a node:

```cpp
auto location() const -> operator_location override {
  return operator_location::remote;
}
```